controls/roles/manage-service/tasks/main.yml

```diff
@@ -36,7 +36,7 @@
       state: directory
       owner: "{{ stereum_service_configuration.user }}"
       group: "2000"
-      mode: 0700
+      mode: 0700  
```

Comment:
please remove trailing spaces, lint doesn't like them

Reply:
it is removed